src/Http/Routing/src/SuppressLinkGenerationMetadata.cs

```diff
@@ -1,16 +1,25 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
+
 namespace Microsoft.AspNetCore.Routing;
 
 /// <summary>
 /// Represents metadata used during link generation. If <see cref="SuppressLinkGeneration"/> is <c>true</c>
 /// the associated endpoint will not be used for link generation.
 /// </summary>
+[DebuggerDisplay("{ToString(),nq}")]
 public sealed class SuppressLinkGenerationMetadata : ISuppressLinkGenerationMetadata
 {
     /// <summary>
     /// Gets a value indicating whether the assocated endpoint should be used for link generation.
     /// </summary>
     public bool SuppressLinkGeneration => true;
+
+    /// <inheritdoc/>
+    public override string ToString()
+    {
+        return $"SuppressLinkGeneration: {SuppressLinkGeneration}";
```

Comment:
It won't ever be false. Same for SuppressMatchingMetadata 
```suggestion
        return "SuppressLinkGeneration: true";
```

Reply:
I don't love hard-coding the value in two places.  Would it just be for the perf win?

Reply:
Technically it should be `SuppressLinkGeneration: True` (capital T).

I don't care about perf here at all. I'd prefer to keep it understandable: here is a property name and here is its property value.